tests/parser/types/numbers/test_constants.py

```diff
@@ -175,11 +175,12 @@ def test_constant_folds(search_for_sublist):
 def test() -> uint256:
     # calculate some constant which is really unlikely to be randomly
     # in bytecode
-    return 2**SOME_CONSTANT * SOME_PRIME
+    ret: uint256 = 2**SOME_CONSTANT * SOME_PRIME
+    return ret
     """
 
     lll = compile_code(code, ["ir"])["ir"]
-    assert search_for_sublist(lll, ["mstore", [0], [2 ** 12 * some_prime]])
+    assert search_for_sublist(lll, ["mstore", [320], [2 ** 12 * some_prime]])
```

Comment:
What was the change here that leads to using 320?

Reply:
Before, for single-word types, the space 0-32 was overwritten and used for the return buffer. The new code doesn't have a special case for single-word types, it always allocates a buffer using the memory allocator, which in this case starts at offset 320. I judged the single byte of savings in the bytecode was not worth the extra case in the codebase. We can revisit that, though.